core/arch/arm/plat-vexpress/tpm2_platform.h

```diff
@@ -0,0 +1,16 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2022, Linaro Limited
+ *
```

Comment:
This line isn't needed, please remove it here and in the other files in the patch.